modules/op-enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,56 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/creating-applications-with-cicd-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="enabling-monitoring-of-event-listeners-for-triggers-for-user-defined-projects_{context}"]
+= Enabling monitoring of event listeners for Triggers for user-defined projects
+
+As a cluster administrator, to gather event listener metrics for the `Triggers` service in a user-defined project and display them in the {product-title} web console, you can create a service monitor for each event listener. On receiving a HTTP request, event listeners for the `Triggers` service return three metrics -- `http_duration_seconds`, `event_count`, and `triggered_resources`.
```

Comment:
@savitaashture only the metrics name `triggered_resources` is enough or we need to add `eventlistener_triggered_resources`?

Reply:
I also had the same doubt - there was a mismatch between the text and the screenshot in Savita's [draft document](https://docs.google.com/document/d/1uumFX8iHEGjSuPjMdNuB4x3ueyuXrOR2TKhLUTXo--0/edit#heading=h.n09pyxcib1k7).

Reply:
Oh sorry yes its `eventlistener_triggered_resources` the one from screenshot 
For reference [follow](https://github.com/tektoncd/triggers/blob/main/docs/eventlisteners.md#configuring-metrics-for-eventlisteners)

Reply:
what about `http_duration_seconds` and `event_count`?

Reply:
yes for all `eventlistener_` is prefixed 
I have updated [draft doc](https://docs.google.com/document/d/1uumFX8iHEGjSuPjMdNuB4x3ueyuXrOR2TKhLUTXo--0/edit#) and also to follow from upstream doc here is the [reference](https://github.com/tektoncd/triggers/blob/main/docs/eventlisteners.md#configuring-metrics-for-eventlisteners)